pkg/controller/container-runtime-config/container_runtime_config_controller_test.go

```diff
@@ -328,6 +332,178 @@ func TestContainerRuntimeConfigCreate(t *testing.T) {
 	}
 }
 
+func TestContainerRuntimeConfigUpdate(t *testing.T) {
```

Comment:
nit: If possible it would be great to have a quick golang doc blurb explaining each test for those reading through.

Reply:
fixed